app/main/presenters/service_presenters.py

```diff
@@ -74,6 +81,18 @@ def get_service_id(self, service_data):
         else:
             return list(chunk_string(str(id), 4))
 
+    def get_external_framework_url(self, service_data):
```

Comment:
I thought we'd got rid of this?

Reply:
I think something's gone wrong with a rebase somewhere - the deleted stuff from https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/1040 has crept back in

Reply:
Whoops - should be sorted now